Makefile

```diff
@@ -0,0 +1,227 @@
+# VERSION defines the project version for the bundle.
+# Update this value when you upgrade the version of your project.
+# To re-generate a bundle for another specific version without changing the standard setup, you can:
+# - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
+# - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
+VERSION ?= 0.0.1
```

Comment:
Yes if we plan to release the operator when new a new chart is released, regardless of its correctness. But I'd be careful about that since we would be risking releasing a version that contains a chart that may not work with the operator before we have tested it.

A reason for not coupling them is that there might be improvements to the helm operator (update the base container image for instance) not tied to the helm chart that would require a release of the chart without changes just to align with the version in the operator.